quizbizz/src/pages/quizanswers/quizquestionanswers/QuizQuestionAnswer.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+const QuizQuestionAnswer = ({ question, correctAnswer, answerSelected }) => {
+  const options = question?.options;
+  return (
+    <div className="quiz-question-answer-container">
+      <h2 className="primary-text-color font-medium-large weight-bold">
+        {`${question?.questionNumber}. ${question?.question}`}
+      </h2>
+      <div className="spacer-2"></div>
+      <ol className="quiz-questions-container list-style-upper-alphabets">
+        <li className="font-medium-large primary-text-color">{options?.[0]}</li>
```

Comment:
Suggestion: as far as I know these should be radio buttons

Reply:
But this is also working fine